app/assets/javascripts/char-count.js

```diff
@@ -81,21 +81,27 @@
           var wrapper = CharCount.prototype.wrapElement(countElement, 'govuk-c-charcount__wrapper')
           var elementId = countElement.getAttribute('id')
           var countHighlightClass = (countElement.type === 'text') ? 'govuk-c-charcount__highlight-input' : 'govuk-c-charcount__highlight'
-          wrapper.insertAdjacentHTML('afterbegin', '<div id="' + elementId + '-hl" class="form-control ' + countHighlightClass + '" aria-hidden="true" role="presentation"></div>')
 
-          var countHighlight = document.getElementById(elementId + '-hl')
-          // countHighlight.style.height = CharCount.prototype.getHeight(countElement) + 'px'
-          // countHighlight.style.height = countElement.getBoundingClientRect().height + 'px'
-          countHighlight.style.height = countElement.offsetHeight + 'px'
-          countHighlight.style.width = countElement.offsetWidth + 'px'
 
-          // We have to disable resize on highlighted components to avoid the async scroll and boundaries
-          countElement.style.resize = 'none'
+          if(document.getElementById(elementId + '-hl') == null) {
+            wrapper.insertAdjacentHTML('afterbegin', '<div id="' + elementId + '-hl" class="form-control ' + countHighlightClass + '" aria-hidden="true" role="presentation"></div>')
```

Comment:
@Muhammad-Ali-Ahmad I think it might just be line 87 we need to wrap in a conditional, though I'm not sure from reading the other code that will be skipped if that code is actually doing anything useful in the context of an accessibility test and that's the only time this will trigger so might be fine as it is 👍